src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Super class of address, email and phone
```

Comment:
need not mention this is the super class of ... 
only we know that we extracted the common stuff out to form a super class; any other developer using this in the client code need not know that ;-)